sdk/CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 card.io Android SDK change log and release notes
 ================================================
 
+5.5.1-OS1
+----
+* Support 16KB page size devices - [PR #1](https://github.com/OutSystems/card.io-Android-source/pull/1)
+
+----
+----
+----
```

Comment:
Do we need all of these lines?

Reply:
Huh Went a bit overboard there didn't I? XD

Reply:
Done in e21b7d394005ebe39cb400aa326864a3d81e48a1